src/app/components/utility/modal/modal.component.ts

```diff
@@ -242,6 +272,48 @@ export class ModalComponent implements OnInit {
       this.isDisabled = e.target.value === this.user.last_name;
     } else if (e.target.name === 'update_affiliation') {
       this.isDisabled = e.target.value === this.user.affiliation;
+    } else if (e.target.name === 'update_google_scholar_url') {
+      if (this.validURL(e.target.value) || e.target.value === '') {
+        this.isDisabled = false;
```

Comment:
@drepram Similarly for `isDisabled`, it will enable the button if we entering the field correctly at the last even though the error message is showing for the entered invalid URL. So how about if we move this `isDisabled` and set to true at the last of `validateModalInput` function with the conditions like checking if `invalidFields` array is not empty and also covering other cases too if there are any.